packages/relay/src/logsBloomUtils.ts

```diff
@@ -0,0 +1,57 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { keccak256 } from 'ethers';
+import { prepend0x, strip0x } from './formatters';
+import { EthImpl } from './lib/eth';
+
+export class LogsBloomUtils {
+  public static readonly BYTE_SIZE = 256;
+  public static readonly MASK = 0x7ff;
+
+  /**
+   * Generate logs bloom for synthetic transaction
+   * @param address
+   * @param topics
+   */
+  public static buildLogsBloom(address: string, topics: string[]): string {
+    if (!address?.length) {
+      return EthImpl.emptyBloom;
+    }
+    if (!topics.length) {
+      return EthImpl.emptyBloom;
+    }
```

Comment:
Let's add a log here for better traceability, in case we have a bug somewhere:
```suggestion
    if (!address?.length) {
      log.warning(`Invalid address passed: ${address}. Returning empty bloom.`);
      return EthImpl.emptyBloom;
    }
    if (!topics?.length) {
      log.warning(`Invalid topics passed: ${topics}. Returning empty bloom.`);
      return EthImpl.emptyBloom;
    }
```

Reply:
The entire project uses a Pino logger. If we want to log a warning here, we must pass the logger from a level above or we should initialize it in that utility class. Also, we can't do it in the constructor because `buildLogsBloom` is a static method. I don't think the effort is worth it but if you insist on adding these warning logs here, I'll try to do it in the most clean way.